checkexperiments.py

```diff
@@ -0,0 +1,266 @@
+                    replicate_obj = replicate_request.json()
+                    if replicate_obj.get('status') not in ['deleted']:
+                        replicates.add(replicate_obj['@id'])
+                        replicates_reads[replicate_obj['@id']] = 0
+                if  ex.get('files'):
+                    for file_acc in ex.get('files'):
+                        file_request = session.get(urljoin(
+                            url,
+                            file_acc + '?frame=object&format=json'))
+                        file_obj = file_request.json()
+                        if file_obj.get('file_format') == 'fastq' and \
+                           file_obj.get('status') not in ['uploading',
+                                                          'content error',
+                                                          'upload failed',
+                                                         ]:
+                            file_date = datetime.datetime.strptime(
+                                file_obj['date_created'][:10], "%Y-%m-%d")
+                            dates.append(file_date)
+                            files.append(file_obj)
+                            if file_obj.get('read_count') and file_obj.get('replicate'):
+                                if not replicates_reads.get(file_obj.get('replicate')) is None:
+                                    replicates_reads[file_obj.get('replicate')] += \
+                                        file_obj.get('read_count')
+        except requests.exceptions.RequestException as e:
+            print (e)
+            continue
+        else:
+            
+            submitted_replicates = set()
+            for file_obj in files:
+                if file_obj.get('replicate'):
+                    submitted_replicates.add(file_obj.get('replicate'))
+            if replicates and not replicates - submitted_replicates:
+                if award_obj.get('rfa') == 'modERN':
+                    err.write(
+                        award_obj.get('rfa') + '\t' +
+                        experiment_accession + '\texcluded from automatic screening\n')
+                    err.flush()
+                else:
+                    # check read depth:
+                    depth_flag = False
+                    if award_obj.get('rfa') == 'modENCODE':
+                        for rep in replicates_reads:
+                            if replicates_reads[rep] < min_depth['modENCODE-chip']:
+                                depth_flag = True
+                                err.write(
+                                    award_obj.get('rfa') + '\t' + \
+                                    experiment_accession + '\t' + rep + \
+                                    '\treads_count=' + str(replicates_reads[rep]) + \
+                                    '\texpected count=' + \
+                                    str(min_depth['modENCODE-chip']) + '\n')
```

Comment:
Use string format.
```
                                msg = (
                                    '{}\t{}\t{}'
                                    '\ttreads_count={}'
                                    '\texpected_count={}\n'.format(
                                        award_obj.get('rfa'),
                                        experiment_accession,
                                        rep,
                                        str(replicates_reads[rep]),
                                        str(min_depth['modENCODE-chip'])
                                    )
                                )
                                err.write(msg)
```
Much easier to read and update.

Reply:
done